### PR TITLE
[docs] Update README

### PR DESCRIPTION
**Pull request by hickeyma**
Updates which would help improve usability of the doc.

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
     <a href="CONTRIBUTING.md">Getting Involved</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="https://gitter.im/open-telemetry/opentelemetry-service">Getting In Touch</a>
+    <a href="https://app.slack.com/client/T08PSQ7BQ/CJFCJHG4Q">Getting In Touch</a>
```
Comment (jpkrohling) [inline README.md:9]: This should probably be https://cloud-native.slack.com/archives/C01N6P7KR6W (#otel-collector) instead of #opentelemetry.

**Review by jpkrohling — COMMENTED** [review]
Thank you for this PR!

File: README.md (hunk 1 above)
Comment (bogdandrutu) [inline README.md:9]:
```suggestion
    <a href="https://cloud-native.slack.com/archives/C01N6P7KR6W">Getting In Touch</a>
```

Comment (bogdandrutu) [reply README.md:9]: Adding this as suggestion.

**Comment by hickeyma** [comment]
Thanks for the reviews and getting this merged @jpkrohling @bogdandrutu

**Pull request merged**